decidim-core/app/views/decidim/pages/index.html.erb

```diff
@@ -7,84 +7,68 @@ edit_link(
+
+  <div class="text-center py-10">
+    <h1 class="h1 decorator mb-10 inline-block">
+      <%= t "decidim.pages.index.title" %>
+    </h1>
+    <p class="text-lg text-gray-2">
+      <%= t("decidim.pages.index.subheading", name: current_organization.name) %>
+    </p>
   </div>
 
-  <% if @topics.any? %>
-    <div class="row">
-      <h2 class="section-heading"><%= t ".topics" %></h2>
-      <div class="row collapse">
-        <div class="row small-up-1 smallmedium-up-2 mediumlarge-up-2 large-up-3 card-grid">
-          <% @topics.each do |topic| %>
+  <div class="space-y-16 mb-36">
+    <% if @topics.any? %>
+      <div class="space-y-6">
+        <h2 class="h2 decorator"><%= t "decidim.pages.index.topics" %></h2>
+
+        <div class="space-y-6" data-component="accordion">
+          <% @topics.each_with_index do |topic, ix| %>
             <% if topic.pages.any? %>
-              <div class="column">
-                <div class="card card--process card--mini">
-                  <div class="card__content">
-                    <h3 class="card__title">
-                      <%= link_to translated_attribute(topic.title), page_path(topic.pages.first), class: "card__link" %>
+              <div class="page__accordion">
+                <div class="page__accordion-trigger">
+                  <div>
+                    <h3 id="accordion-title-<%= ix %>" class="page__accordion-trigger-title">
+                      <%= link_to translated_attribute(topic.title), page_path(topic.pages.first) %>
                     </h3>
-                    <p><%= translated_attribute topic.description %></p>
-                    <ul>
-                      <% topic.pages.each do |page| %>
-                        <li><%= link_to translated_attribute(page.title), page_path(page) %></li>
-                      <% end %>
-                    </ul>
-                  </div>
-                  <div class="card__footer">
-                    <div class="card__support">
-                      <%= link_to page_path(topic.pages.first),
-                                  class: "card__button button button--sc small" do %>
-                        <%= t(".read_more") %>
-                        <span class="show-for-sr"><%= translated_attribute(topic.title) %></span>
-                      <% end %>
-                    </div>
+                    <p class="page__accordion-trigger-snippet"><%= translated_attribute topic.description %></p>
                   </div>
+                  <button id="accordion-trigger-<%= ix %>" data-controls="accordion-panel-<%= ix %>" aria-labelledby="accordion-title-<%= ix %>">
+                    <%= icon "arrow-down-s-line", class: "page__accordion-trigger-arrow" %>
+                    <%= icon "arrow-up-s-line", class: "page__accordion-trigger-arrow" %>
+                  </button>
+                </div>
+
+                <div id="accordion-panel-<%= ix %>" class="page__accordion-panel">
```

Comment:
This is bothering that the accordion panel "flashes" on the screen when it is initially loaded. It is initially visible and then it is closed once the JS runs on the page:

https://user-images.githubusercontent.com/864340/183084008-22682e2c-0b5d-4bce-9603-29b76592ac9d.mp4

I would suggest to hide it in the initial state (if it's supposed to be hidden initially) and then show it after the element is triggered. If you want to make it work also for non-JS clients, add special CSS that shows it for non-JS clients.

Reply:
Solved. It was about a missing `aria-hidden="true"` on the element itself. Previously, it was the a11y lib who enforced such setting, therefore from the initial load to the script execution there was a timeframe where the element remained visible